spec.md

```diff
@@ -182,18 +183,19 @@ In order to verify that a repository contains a given manifest or blob, make a `
 
 `/v2/<name>/blobs/<digest>` <sup>[end-2](#endpoints)</sup> (for blobs).
 
-A HEAD request to an existing blob or manifest URL MUST return `200 OK`. A successful response SHOULD contain the digest
-of the uploaded blob in the header `Docker-Content-Digest`.
+A HEAD request to an existing blob or manifest URL MUST return `200 OK`.
+A successful response SHOULD contain the digest of the uploaded blob in the header `Docker-Content-Digest`.
 
 If the blob or manifest is not found in the registry, the response code MUST be `404 Not Found`.
 
 #### Push
 
-Pushing an artifact typically works in the opposite order as a pull: the blobs making up the artifact are uploaded first,
-and the manifest last. Strictly speaking, content can be uploaded to the registry in any order, but a registry MAY reject
-a manifest if it references blobs that are not yet uploaded, resulting in a `BLOB_UNKNOWN` error <sup>[code-1](#error-codes)</sup>.
+Pushing an artifact typically works in the opposite order as a pull: the blobs making up the artifact are uploaded first, and the manifest last.
 A useful diagram is provided [here](https://github.com/google/go-containerregistry/tree/d7f8d06c87ed209507dd5f2d723267fe35b38a9f/pkg/v1/remote#anatomy-of-an-image-upload).
 
+A registry MAY reject a manifest of any type uploaded to the manifest endpoint if it references blobs that do not exist in the registry.
```

Comment:
To cover the index/manifest list, I think we should be clear this applies to manifest digests in addition to blob digests.

```suggestion
A registry MAY reject a manifest of any type uploaded to the manifest endpoint if it references manifests or blobs that do not exist in the registry.
```

Reply:
Thanks, change added.